core/src/main/java/eu/tsystems/mms/tic/testframework/report/model/context/SessionContext.java

```diff
@@ -130,6 +131,14 @@ public void setUserAgent(String userAgent) {
         this.userAgent = userAgent;
     }
 
+    public String getBaseUrl() {
```

Comment:
The `WebDriverRequest` already has `baseUrl` property. So this seems redundant.

Reply:
Well, `baseUrl` is defined in `SeleniumWebDriverRequest` which is part of the module `driver-ui`. But `ContextExporter` has only access to `core`.
To prevent the definition in `SessionContext`, I can move get/set of `baseUrl` to the interface `WebdriverRequest`, because browser information are already part of it.